### PR TITLE
VictorK react-toggle-switch-component (1st submission)

### PR DESCRIPTION
**Pull request by vkalinkin**
toggle 1
toggle 2

### CODE REVIEW

File: react-toggle-switch-component/src/toggle-switch.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    } else {
+      this.setState({ isOn: true });
+    }
+  }
+
+  render() {
+    let className = 'toggle off';
+    let spanText;
+    let circleName;
+    if (this.state.isOn) {
```
Comment (Zhiwutian) [inline react-toggle-switch-component/src/toggle-switch.jsx:22]: The only update I would do here is move the logic into a different method and have that method return an object with the className, circleName, and spanText values so you can use that object to set the values in the JSX.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Very nice work Victor, a small update to polish the render method!

**Comment by vkalinkin** [comment]
toggle 3

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**